src/modules/asab-config/configuration.css

```diff
@@ -4,14 +4,24 @@
 	height: 100%;
 }
 
-.library-row {
+.config-row {
  	height: 100%;
 }
 
-.library-container {
+.config-container {
```

Comment:
Ok ... and will library view work then?

Reply:
Sure, it will, since this was taken one-to-one from the library container of LM.io WebUI and it was reused in ASAB-WebUI's ASAB-config module. I just renamed it here not to confuse people